libs/ui/src/lib/datepicker/datepicker-calendar.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="cvi-datepicker__calendar" tabindex="-1" id="datePickerCalendar">
+  <div class="cvi-datepicker__calendar-header">
+    <div class="cvi-datepicker__calendar-header-chevron">
+      <button aria-label="Previous month" (click)="previousMonth()">
+        <svg
+          class="cvi-datepicker__header-chevron-icon"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24" width="24" height="24">
+          <path d="M14.83 2.83L12 0L0 12L12 24L14.83 21.17L5.66 12L14.83 2.83Z" fill="white"/>
+        </svg>
+      </button>
+    </div>
+    <div *ngIf="this.displayDate"
+         id="monthYearLabel"
+         class="cvi-datepicker__calendar-header-month">{{ monthLabels[displayDate.getMonth()] }} {{ displayDate.getFullYear() }}</div>
+    <div class="cvi-datepicker__calendar-header-chevron">
+      <button aria-label="Next month" (click)="nextMonth()">
+        <svg
+          class="cvi-datepicker__header-chevron-icon"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24" width="24" height="24">
+          <path d="M2.99992 0L0.169922 2.83L9.33992 12L0.169922 21.17L2.99992 24L14.9999 12L2.99992 0Z" fill="white"/>
+        </svg>
+      </button>
+    </div>
+  </div>
+  <div role="grid" aria-labelledby="monthYearLabel" class="cvi-datepicker__calendar-grid">
```

Comment:
Sounds reasonable! But this might take a bit more time to implement - let's see if we have enough time in this dev round. If not then a separate ticket can be created.

Reply:
Resolving this to be implemented later.